### PR TITLE
Updated Security Model and FAQ pages

### PR DESCRIPTION
**Pull request by jsosulska**
Documentation update around the security model.

### CODE REVIEW

File: website/source/docs/faq.html.md [hunk 1]
```diff
@@ -105,3 +105,16 @@ available, but that won't otherwise affect the local datacenter. There are some
 situations where a limited subset of data can be replicated, such as with Consul's built-in
 [ACL replication](https://learn.hashicorp.com/consul/day-2-operations/acl-replication) capability, or
 external tools like [consul-replicate](https://github.com/hashicorp/consul-replicate).
+
+## Q: Can Consul natively handle protecting against other processes accessing Consul's memory state? 
+
+In short, Consul does not have anything built in to protect against memory 
+viewing doesn't interact with the host system to change or manipulate
```
Comment (preetapan) [inline website/source/docs/faq.html.md]: missing an "and" here?

File: website/source/docs/internals/security.html.md [hunk 2]
```diff
@@ -36,6 +36,18 @@ any of the settings below are not enabled, then parts of this threat model are
...
+  binary and obeys the same security requirements as any other application on 
+  your system. Consul doesn't interact with the host system to change or 
+  manipulate security values in any way. Take any precautions or remediation 
+  steps that you would normally do for individual processes, based on your 
+  operating system. 
+  Some example remediation steps you could take are outlined below.
+  - Run applications, including Consul, as non-root users with appropriate 
+  configurations
+  - Run a secure kernel module, such as the SELinux 
+  - Secure against unprivledged users becoming root
```
Comment (preetapan) [inline website/source/docs/internals/security.html.md]: typo, should be unprivileged

**Comment by jsosulska** [comment]
Addressed your changes @preetapan !
Thank you so much :)

File: website/source/docs/faq.html.md [hunk 3]
```diff
@@ -105,3 +105,16 @@ available, but that won't otherwise affect the local datacenter. There are some
...
 [ACL replication](https://learn.hashicorp.com/consul/day-2-operations/acl-replication) capability, or
 external tools like [consul-replicate](https://github.com/hashicorp/consul-replicate).
+
+## Q: Can Consul natively handle protecting against other processes accessing Consul's memory state? 
+
+In short, Consul does not have anything built in to protect against memory 
+viewing and doesn't interact with the host system to change or manipulate 
+application security.
+
+As Consul is a singular binary, we recommend taking any precautions or
```
Comment (blake) [inline website/source/docs/faq.html.md]:
```suggestion
We recommend taking any precautions or 
```

File: website/source/docs/internals/security.html.md [hunk 4]
```diff
@@ -36,6 +36,18 @@ any of the settings below are not enabled, then parts of this threat model are
...
+* **Consul runs just like any other binary.** Consul runs as a single process 
+  binary and obeys the same security requirements as any other application on 
+  your system. Consul doesn't interact with the host system to change or 
+  manipulate security values in any way. Take any precautions or remediation 
+  steps that you would normally do for individual processes, based on your 
+  operating system. 
+  Some example remediation steps you could take are outlined below.
+  - Run applications, including Consul, as non-root users with appropriate 
+  configurations
+  - Run a secure kernel module, such as the SELinux
```
Comment (blake) [inline website/source/docs/internals/security.html.md]:
```suggestion
  - Implement Mandatory Access Control using a kernel security module such as SELinux 
```

File: website/source/docs/faq.html.md [hunk 5]
```diff
@@ -105,3 +105,16 @@ available, but that won't otherwise affect the local datacenter. There are some
...
+In short, Consul does not have anything built in to protect against memory 
+viewing and doesn't interact with the host system to change or manipulate 
+application security.
+
+We recommend taking any precautions or 
+remediation steps that you would normally do for individual processes, based 
+on your operating system. 
+
+Please see our 
+[Security Model](https://www.consul.io/docs/internals/security.html) for more.
```
Comment (blake) [inline website/source/docs/faq.html.md]:
```suggestion
[Security Model](https://www.consul.io/docs/internals/security.html) for more information.
```

File: website/source/docs/faq.html.md [hunk 6]
```diff
@@ -105,3 +105,16 @@ available, but that won't otherwise affect the local datacenter. There are some
 situations where a limited subset of data can be replicated, such as with Consul's built-in
 [ACL replication](https://learn.hashicorp.com/consul/day-2-operations/acl-replication) capability, or
 external tools like [consul-replicate](https://github.com/hashicorp/consul-replicate).
+
+## Q: Can Consul natively handle protecting against other processes accessing Consul's memory state? 
+
+In short, Consul does not have anything built in to protect against memory
```
Comment (blake) [inline website/source/docs/faq.html.md]:
```suggestion
Consul does not provide built-in memory access protections, and doesn't interact with the host system to change or manipulate 
```

File: website/source/docs/internals/security.html.md [hunk 7]
```diff
@@ -36,6 +36,18 @@ any of the settings below are not enabled, then parts of this threat model are
 going to be invalid. Additional security precautions must also be taken for
 items outside of Consul's threat model as noted in sections below.
 
+* **Consul runs just like any other binary.** Consul runs as a single process 
+  binary and obeys the same security requirements as any other application on
```
Comment (blake) [inline website/source/docs/internals/security.html.md]:
```suggestion
* **Consul runs just like any other binary.** Consul runs as a single process 
  and obeys the same security requirements as any other application on 
```

**Pull request merged**